### ISSUE TITLE
Signature extraction failed

### ISSUE THREAD

**Issue by prateeknischal** [rg3/youtube-dl#11678]
Version : fresh install on 
`Linux prateek 4.4.0-57-generic #78-Ubuntu SMP Fri Dec 9 23:50:32 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux`

`youtube-dl --version`
2017.01.08

Video : https://www.youtube.com/watch?v=I8kPYTwOrXY

`$ youtube-dl https://www.youtube.com/watch?v=I8kPYTwOrXY --verbose`
The above youtube link causes a Signature extraction failed.

Trace : [paste](https://paste.ubuntu.com/23782714/)

**Comment by matoro** [comment]
Confirming this also happening on https://www.youtube.com/watch?v=1Bs6vdD1aYc, just build from git.

**Comment by yan12125** [comment]
@prateeknischal Already fixed in 2017.01.10. 2017.01.08 is broken.

@matoro Could you run ```youtube-dl -v --write-pages 1Bs6vdD1aYc``` and paste the output as well as upload all *.dump files?

**Comment by AmarOk1412** [comment]
Same problem with with youtube-dl on Fedora 25 (installed via dnf (2016.12.22)).

```python
youtube-dl -v --write-pages 1Bs6vdD1aYc
[debug] System config: ['--prefer-free-formats']
[debug] User config: []
[debug] Command-line args: ['-v', '--write-pages', '1Bs6vdD1aYc']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2016.12.22
[debug] Python version 3.5.2 - Linux-4.8.15-300.fc25.x86_64-x86_64-with-fedora-25-Twenty_Five
[debug] exe versions: ffmpeg 3.1.6, ffprobe 3.1.6
[debug] Proxy map: {}
[youtube] 1Bs6vdD1aYc: Downloading webpage
[youtube] Saving request to 1Bs6vdD1aYc_https_-_www.youtube.com_watchv=1Bs6vdD1aYc_gl=US_hl=en_has_verified=1_bpctr=9999999999.dump
[youtube] 1Bs6vdD1aYc: Downloading video info webpage
[youtube] Saving request to 1Bs6vdD1aYc_https_-_www.youtube.com_get_video_info_video_id=1Bs6vdD1aYc_el=info_ps=default_eurl=_gl=US_hl=en.dump
[youtube] 1Bs6vdD1aYc: Extracting video information
[youtube] {22} signature length 44.40, html5 player en_US-vflsagga9
[youtube] 1Bs6vdD1aYc: Downloading player https://s.ytimg.com/yts/jsbin/player-en_US-vflsagga9/base.js
[youtube] Saving request to 1Bs6vdD1aYc_https_-_s.ytimg.com_yts_jsbin_player-en_US-vflsagga9_base.js.dump
ERROR: Signature extraction failed: Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 1005, in _decrypt_signature
    video_id, player_url, s
  File "/usr/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 932, in _extract_signature_function
    cache_res = res(test_string)
  File "/usr/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 984, in <lambda>
    return lambda s: initial_function([s])
  File "/usr/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 254, in resf
    res, abort = self.interpret_statement(stmt, local_vars)
  File "/usr/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 55, in interpret_statement
    v = self.interpret_expression(expr, local_vars, allow_recursion)
  File "/usr/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 174, in interpret_expression
    return obj[member](argvals)
KeyError: 'qC'
 (caused by KeyError('qC',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 1005, in _decrypt_signature
    video_id, player_url, s
  File "/usr/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 932, in _extract_signature_function
    cache_res = res(test_string)
  File "/usr/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 984, in <lambda>
    return lambda s: initial_function([s])
  File "/usr/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 254, in resf
    res, abort = self.interpret_statement(stmt, local_vars)
  File "/usr/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 55, in interpret_statement
    v = self.interpret_expression(expr, local_vars, allow_recursion)
  File "/usr/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 174, in interpret_expression
    return obj[member](argvals)
KeyError: 'qC'
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 1005, in _decrypt_signature
    video_id, player_url, s
  File "/usr/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 932, in _extract_signature_function
    cache_res = res(test_string)
  File "/usr/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 984, in <lambda>
    return lambda s: initial_function([s])
  File "/usr/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 254, in resf
    res, abort = self.interpret_statement(stmt, local_vars)
  File "/usr/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 55, in interpret_statement
    v = self.interpret_expression(expr, local_vars, allow_recursion)
  File "/usr/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 174, in interpret_expression
    return obj[member](argvals)
KeyError: 'qC'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/youtube_dl/YoutubeDL.py", line 693, in extract_info
    ie_result = ie.extract(url)
  File "/usr/lib/python3.5/site-packages/youtube_dl/extractor/common.py", line 358, in extract
    return self._real_extract(url)
  File "/usr/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 1584, in _real_extract
    encrypted_sig, video_id, player_url, age_gate)
  File "/usr/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 1015, in _decrypt_signature
    'Signature extraction failed: ' + tb, cause=e)
youtube_dl.utils.ExtractorError: Signature extraction failed: Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 1005, in _decrypt_signature
    video_id, player_url, s
  File "/usr/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 932, in _extract_signature_function
    cache_res = res(test_string)
  File "/usr/lib/python3.5/site-packages/youtube_dl/extractor/youtube.py", line 984, in <lambda>
    return lambda s: initial_function([s])
  File "/usr/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 254, in resf
    res, abort = self.interpret_statement(stmt, local_vars)
  File "/usr/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 55, in interpret_statement
    v = self.interpret_expression(expr, local_vars, allow_recursion)
  File "/usr/lib/python3.5/site-packages/youtube_dl/jsinterp.py", line 174, in interpret_expression
    return obj[member](argvals)
KeyError: 'qC'
 (caused by KeyError('qC',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

**Comment by matoro** [comment]
@yan12125 Fixed after another rebuild.  Thanks for quick response :smile:

**Comment by prateeknischal** [comment]
`youtube-dl` in `apt-get` for the above `uname` (linux Xubuntu 16.04) didn't provide builds after 2017.01.08, fixed in `2017.01.10`
